### ISSUE TITLE
No support for nested video directories

### ISSUE THREAD

**Issue by jgm14** [subramanya1997/download_echo360#5]
When a course contains nested subdirectories of videos (for example, nested by week). The script cannot traverse the directories recursively to get the videos,

**Comment by subramanya1997** [comment]
@jgm14  yes. I graduated and I don't have access to my echo account anymore. Have kinda stopped developing. Feel free to fork it and add in features.
